pykanka/child_types.py

```diff
@@ -115,7 +86,7 @@ def post(self, json_data: str = None, **kwargs):
         :return: requests.response
         """
 
-        payload, files = self._prepare_post(json_data, **kwargs)
+        payload, files = self.client._prepare_post(json_data, **kwargs)
```

Comment:
_prepare_post is method of GenericChildType, not KankaClient, so .client should be removed